File: src/main/java/com/dj/tool/model/CommentTableModel.java
package com.dj.tool.model;

import javax.swing.table.DefaultTableModel;


public class CommentTableModel extends DefaultTableModel {

    public CommentTableModel(Object[][] data, Object[] columnNames) {
        super(data, columnNames);
    }

    @Override
    public boolean isCellEditable(int row, int column) {
        if (column > 0 && column < 7) {
            return true;
        }
        return false;
    }


}

File: src/main/kotlin/com/dj/tool/model/CommentTableModel.kt
package com.dj.tool.model

import javax.swing.table.DefaultTableModel

class CommentTableModel(data: Array<Array<Any?>>, columnNames: Array<Any?>?) : DefaultTableModel(data, columnNames) {
    override fun isCellEditable(row: Int, column: Int): Boolean {
        if (column > 0 && column < 7) {
            return true
        }
        return false
    }
}
